src/arrayMethodSort.js

```diff
@@ -3,9 +3,41 @@
 /**
  * Implement method Sort
  */
+const compareStrings = (a, b) => {
+  const strA = String(a);
+  const strB = String(b);
+
+  if (strA > strB) {
+    return 1;
+  }
+
+  if (strA === strB) {
+    return 0;
+  }
+
+  return -1;
+};
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareStrings) {
+    let count;
+
+    do {
```

Comment:
Better to use `for` cycle, then you dont need to create additional variable, and your code gets cleaner and more readible. Also read some info about simplest sorting methods:

1)[ Bubble sort](https://www.geeksforgeeks.org/bubble-sort/)
2) [Selection sort](https://www.geeksforgeeks.org/selection-sort/)